modules/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/MaxmindIpDataLookups.java

```diff
@@ -116,6 +118,11 @@ static class AnonymousIp extends AbstractBase<AnonymousIpResponse> {
             );
         }
 
+        @Override
+        protected AnonymousIpResponse record(AnonymousIpResponse response) {
```

Comment:
I kind of wonder if this method name is going to cause confusion for future maintainers, since as you've pointed out your first thought when you see this is the verb -- what am I recording? What about `cacheableRecord`?

Reply:
Indeed, my unwavering love of short names is definitely at odds with the desire for a clear name in this case. I renamed it to `cacheableRecord` for now via 9dd8964, but I think I'm going to polish things a little more (on subsequent PRs).